### DISCUSSION TITLE
Can we take delta of delta incremental backup?

### DISCUSSION THREAD

**Discussion by hanamakk** [Altinity/clickhouse-backup#1056] [General]
Let's say we have taken Monday's backup as Incremental with respect to Sunday's full backup. Is it possible to take incremental backup for Tuesday using diff-from-remote as Monday backup?

The reason why we are asking is, full backup is taking around 3 hr and each incremental backup takes ~40min. This is causing lot of CPU load and response time of ClickHouse is increasing at the time of taking backup.

Will the dedicated user for backup jobs will help reduce the load on ClickHouse at the time of taking backup?

**Comment by Slach** [comment]
> Is it possible to take incremental backup for Tuesday using diff-from-remote as Monday backup?

Yes
this is possible

but try to avoid too long incremental backup sequence, 7 is enough
look examples here https://github.com/Altinity/clickhouse-backup/blob/master/Examples.md#how-to-use-clickhouse-backup-in-kubernetes


increment size depends on how many mutations and merges happens from previous backup creation time and how man new data parts was created
